manifests/50-operator-ibm-cloud-managed.yaml

```diff
@@ -61,10 +61,10 @@ spec:
           name: trusted-ca
         - mountPath: /apiserver.local.config/certificates
           name: apiservice-cert
+      hostUsers: false
       priorityClassName: system-cluster-critical
       securityContext:
         runAsNonRoot: true
-        runAsUser: 499
```

Comment:
This user is somehow arbitrarily created in the Dockerfile, I could remove that as well...